src/Cli/dotnet/commands/dotnet-list/dotnet-list-package/ListPackageReferencesCommand.cs

```diff
@@ -16,7 +16,13 @@ internal class ListPackageReferencesCommand : CommandBase
         public ListPackageReferencesCommand(
             ParseResult parseResult) : base(parseResult)
         {
-            _fileOrDirectory = parseResult.GetValue(ListCommandParser.SlnOrProjectArgument);
+            _fileOrDirectory = GetAbsolutePath(Directory.GetCurrentDirectory(),
+                parseResult.GetValue(ListCommandParser.SlnOrProjectArgument));
+        }
+
+        private static string GetAbsolutePath(string currentDirectory, string relativePath)
+        {
+            return Path.GetFullPath(Path.Combine(currentDirectory, relativePath));
```

Comment:
I'm not sure if it's a real scenario or not, but wanted to point out that this `Path.Combine` will introduce a new possible exception. It will throw `ArgumentException` if `SlnOrProjectArgument` is null.

Reply:
You are right. But when `SlnOrProjectArgument` is null from the user, we default to the current directory internally so we should be ok.